website/docs/d/app_token.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
In order to be accessible, this file needs to be linked appropriately in the github.erb file.

Reply:
Created a link in the file. Do I need to generate a proper HTML file?

Reply:
Nope! You're all good; that's done on Hashicorp's side.